src/test-helpers/e2e-helpers/mobile/gestures/newThoughtGesture.ts

```diff
@@ -0,0 +1,18 @@
+import { Browser } from 'webdriverio'
+
+/** New thought gesture. */
+const newThoughtGesture = (browser: Browser<'async'>) => {
+  return browser.touchPerform(
```

Comment:
That's so awesome that we can execute touch gestures!!!

Reply:
Yesss! 🥳